allennlp/commands/bulk.py

```diff
@@ -2,19 +2,22 @@
 from contextlib import ExitStack
 import json
 import sys
-from typing import Sequence, Optional, IO
+from typing import Optional, IO
 
 from allennlp.service.servable import Servable, ServableCollection
 
+def add_bulk_subparser(parser: argparse._SubParsersAction) -> argparse.ArgumentParser:  # pylint: disable=protected-access
+    description = '''Run the specified model against a JSONL input file.'''
```

Comment:
it's json lines, but I guess people don't use JSONL as an acronym, I wrote it out